cmd/datablue/handlers.go

```diff
@@ -360,6 +360,23 @@ func writeText(r *http.Request, ma, pin string, n int) error {
 	return model.WriteText(r.Context(), mediaStore, &model.Text{MID: mid, Timestamp: ts, Data: string(data), Type: tt})
 }
 
+// writeBinary writes binary data.
```

Comment:
Unless this comment is expanded, I don't think it's worth having (unexported functions don't need comments). If we want to keep, it might be worth saying that the data is pulled from the request body and written to the datastore in a Binary entity.